R/db-odbc-mssql.R

```diff
@@ -100,3 +100,45 @@
   )
   DBI::dbExecute(con, sql)
 }
+
+#' @export
+`db_compute.Microsoft SQL Server` <- function(con,
+                                              table,
+                                              sql,
+                                              temporary = TRUE,
+                                              unique_indexes = list(),
+                                              indexes = list(),
+                                              ...)
+{
+  if(!is.list(indexes))
+    indexes <- as.list(indexes)
+
+  if(!is.list(unique_indexes))
+    unique_indexes <- as.list(unique_indexes)
+
+  db_save_query(con, sql, table, temporary = TRUE)
+  db_create_indexes(con, table, unique_indexes, unique=TRUE)
+  db_create_indexes(con, table, indexes, unique=FALSE)
+
+  table
+}
+
+
+#' @export
+`db_save_query.Microsoft SQL Server` <- function(con,
+                                                 sql,
+                                                 name,
+                                                 temporary = TRUE,
+                                                 ...)
+{
+  # check that name has prefixed '##' if temporary
```

Comment:
What's the difference between a prefix of `#` vs `##`?

Reply:
A `#` creates a local temp table, and `##` creates a global temp table.  I'm trying to recall why I choose global,  I do remember that I tested it both ways, and I think local gave me errors. https://technet.microsoft.com/en-us/library/ms186986(v=sql.105).aspx#Anchor_1

Reply:
See https://github.com/r-dbi/odbc/issues/127 for the upstream issue.